Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/WindowsSpeechInputDeviceManager.cs

```diff
@@ -30,9 +30,16 @@ public class WindowsSpeechInputDeviceManager : BaseDeviceManager, IMixedRealityS
         /// </summary>
         public IMixedRealityInputSource InputSource = null;
 
-
         private KeywordRecognizer keywordRecognizer;
 
+        /// <summary>
+        /// Query whether or not the speech system is active
+        /// </summary>
```

Comment:
/// <inheritdoc />

Reply:
What is this referring to?

Reply:
Was rendering strangely. Updated with code block.

Reply:
Yes, replace the summary here. It's pointing to the interface documentation, so don't replace it there.

Reply:
Sorry, what I meant was: Should I also update the StartRecognition and StopRecognition methods while I'm in this file since they were also a copy/paste from the interface documentation

Reply:
oh yes, thanks :3